src/main/java/io/github/nucleuspowered/nucleus/commands/kit/KitCommand.java

```diff
@@ -0,0 +1,102 @@
+import io.github.nucleuspowered.nucleus.internal.annotations.Modules;
+import io.github.nucleuspowered.nucleus.internal.annotations.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.interfaces.InternalNucleusUser;
+import io.github.nucleuspowered.nucleus.internal.permissions.SuggestedLevel;
+import io.github.nucleuspowered.nucleus.internal.services.datastore.UserConfigLoader;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.Inventory;
+import org.spongepowered.api.text.Text;
+
+import java.util.List;
+
+/**
+ * Allows a user to warp to the specified warp.
+ *
+ * Command Usage: /kit Permission: nucleus.kit.base
+ */
+@Permissions(suggestedLevel = SuggestedLevel.ADMIN)
+@Modules(PluginModule.KITS)
+@RegisterCommand("kit")
+public class KitCommand extends CommandBase<Player> {
+
+    private final String kit = "kit";
+
+    @Inject private KitsConfig kitConfig;
+    @Inject private UserConfigLoader userConfigLoader;
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).children(this.createChildCommands())
+                .arguments(GenericArguments.onlyOne(GenericArguments.string(Text.of(kit)))).build();
+    }
+
+    @Override
+    public CommandResult executeCommand(Player player, CommandContext args) throws Exception {
+        String kitName = args.<String>getOne(kit).get();
+        InternalNucleusUser user = userConfigLoader.getUser(player.getUniqueId());
+
+        if (kitConfig.getKits().contains(kitName)) {
+            Kit kit = kitConfig.getKit(kitName);
+
+            if (kit.getInterval() != 0) {
+                if (user.getKitLastUsedTime().containsKey(kitName)) {
+                    if (user.getKitLastUsedTime().get(kitName) > 0) {
+                        player.sendMessage(Util.getTextMessageWithFormat("command.kit.cooldown", String.valueOf(kit.getInterval()),
+                                kit.getTimeUnit().name().toLowerCase()));
+                        return CommandResult.empty();
+                    } else {
+                        user.getKitLastUsedTime().remove(kitName);
+                        user.getKitLastUsedTime().put(kitName, System.currentTimeMillis());
+                    }
+                } else {
+                    user.getKitLastUsedTime().put(kitName, System.currentTimeMillis());
+                }
+            }
+
+            List<Inventory> slots = Lists.newArrayList(player.getInventory().slots().iterator());
+
+            for (int i = 0; i < slots.size(); i++) {
+                Inventory slot = slots.get(i);
+
+                if (kit.getSlots().get(i) != null)
```

Comment:
I'm going to ask for `{ }` around `if` statements please. One line if statements tend to be error prone - the [goto fail ](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) bug in Apple products was because of this!

(Not saying we're an SSL provider, but you know!)